routers/routes/routes.go

```diff
@@ -523,6 +523,11 @@ func RegisterRoutes(m *macaron.Macaron) {
 				return
 			}
 		})
+		m.Group("/branches", func() {
```

Comment:
@lafriks The tabs are listed by iterate repo's units now. If there is a new unit to identify tab, it will not be visible.